### PR TITLE
Fix windows gitconfig uitests

### PR DESCRIPTION
**Pull request by kdubau**

### CODE REVIEW

**Comment by manish** [comment]
:shipit:

File: main/tests/UserInterfaceTests/Util.cs [hunk 1]
```diff
@@ -39,6 +39,11 @@ public static void PrintData (this object data)
 				TestService.Session.DebugObject.Debug (data.ToString ());
 		}
 
+		public static string ToPathSafeString (this string str)
+		{
+			return str.Replace (@"\", "").Replace ("/", "");
```
Comment (Therzok) [inline main/tests/UserInterfaceTests/Util.cs]: Shouldn't this replaced with dashes?

Comment (Therzok) [inline main/tests/UserInterfaceTests/Util.cs]: You'd end up with `originmaster` strings in the worst case scenario.

Comment (kdubau) [inline main/tests/UserInterfaceTests/Util.cs]: good Idea I'll update it to dashes

**Pull request merged**